src/App.tsx

```diff
@@ -4,32 +4,33 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useRef, useState } from "react";
+import { JSX, useState } from "react";
+import { NewCard } from "./components/NewCard";
 
-function App() {
-  const [cardsAmount, setCardsAmount] = useState(1)
-  const [cards, setCards] = useState([{}]);
+interface AppProps {
+  basicCards?: { face: string; back: string }[];
+}
+function App(props: AppProps) {
+  let basicCards: JSX.Element[] = [];
```

Comment:
**issue:** You should never do that when working with state in react - that's why you have `useState`. Keep initial array there